src/main/java/domain/Table.java

```diff
@@ -1,12 +1,60 @@
 package domain;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class Table {
     private final int number;
+    private final Map<Menu, Integer> orderHistory = new HashMap<>();
 
     public Table(final int number) {
         this.number = number;
     }
 
+    public boolean isSame(String number) {
+        return String.valueOf(this.number).equals(number);
+    }
+
+    public void addOrder(Menu menu, int menuQuantity) {
+        orderHistory.put(menu, menuQuantity);
+    }
+
+    public boolean hasOrder() {
+        return !orderHistory.isEmpty();
+    }
+
+    public int totalPrice() {
+        int totalPrice = 0;
+        for (Menu menu : orderHistory.keySet()) {
+            int orderQuantity = orderHistory.get(menu);
+            int menuPrice = menu.getPrice();
+            totalPrice += orderQuantity * menuPrice;
+        }
+        return totalPrice;
+    }
+
+    public int orderNumberOfChickenCategory() {
+        int numberOfChickenCategory = 0;
+        for (Menu menu : orderHistory.keySet()) {
+            if (menu.isChickenCategory()) {
+                numberOfChickenCategory += orderHistory.get(menu);
+            }
+        }
+        return numberOfChickenCategory;
+    }
+
+    public void clearTable(){
+        orderHistory.clear();
+    }
+
+    public Map<Menu, Integer> getOrderHistory() {
+        return orderHistory;
```

Comment:
```
public static List<Menu> menus() {
        return Collections.unmodifiableList(menus);
    }
```
위 기본 제공 코드에서도 알 수 있듯이, getter를 통해 Collection을 얻을 때는 불변객체를 활용하시는 걸 추천합니다.
객체 참조값이 넘어가는 것이기 때문에 이 getter를 통해 받은 인자를 바꿀 경우 해당 객체 자체가 변경됩니다.
이는 orderHistory의 변경까지 가져오기 때문에 객체의 캡슐화를 해치게 됩니다.
```suggestion
        return Collections.unmodifiableMap(orderHistory);
```